profiles/default/linux/amd64/13.0/desktop/proaudio/gnome/parent

```diff
@@ -0,0 +1 @@
+/usr/portage/profiles/default/linux/amd64/13.0/desktop/gnome
```

Comment:
I don't like this, with a hardcoded path. I figured there must be a better way of doing this, and indeed there is. `man portage` says:

> If layout.conf is new enough, you can also use the \<repo\>:\<path\> syntax. The \<repo\> is the same string as is stored in the repo_name file (or omitted to refer to the current repo), and \<path\> is a subdir starting at  profiles/.

That makes a lot more sense.